depcheck.config.js

```diff
@@ -13,7 +13,7 @@ const GQL_V2_TRANSFORMER_PACKAGES = [
   'amplify-graphql-function-transformer',
   'amplify-graphql-http-transformer',
   'amplify-graphql-index-transformer',
-  'amplify-graphql-maps-to-transformer',
+  'amplify-graphql-name-mapping-transformer',
```

Comment:
Is there a reason why we can't use the existing package name?

Reply:
The NPM package name is still the same, but the folder is renamed to more appropriately convey that it now holds 2 transformers related to name mapping.